.github/ct.yaml

```diff
@@ -3,5 +3,5 @@ remote: origin
 chart-dirs:
   - charts
 chart-repos:
-  - uptime-kuma=https://dirsigler.github.io/uptime-kuma-helm
+  - uptime-kuma=https://chrisbsmith.github.io/uptime-kuma-helm
```

Comment:
This should not change, as the source still remains here in this repository.

Reply:
👍  oops, this is a hold over from my fork